README.md

```diff
@@ -13,7 +13,8 @@ Usage and defaults:
 - name: Get default configuration files for NR static analysis tools
   uses: newrelic/newrelic-infra-checkers@v1
   with:
-    semgrep-append: false # Optional, if set to true, local semgrep file policies will join the ones in this repository. 
+    semgrep-append: false # Optional, if set to true, local semgrep file policies will join the ones in this repository.
+    golangci-lint-limited: true # Optional, if set to true, `golangci-lint-limited` config will be used instead of `golangci-lint`.
```

Comment:
I like the idea, it maybe harder to keep "limited" as default instead of "" but maybe it does worth it.

Something like this?

```yaml
    with:
       golangci-lint-suffix: limited
```

Reply:
Looks good to me, maybe `golangci-lint-config` instead of suffix ? Something like we do with the publish action and the schema files. 

Regarding defaults i think the `action.yml` has some parameters to define defaults for inputs.

Reply:
I changed and I think now it is not only more flexible but also simpler [2563081](https://github.com/newrelic/newrelic-infra-checkers/pull/6/commits/25630817875c9d0f61820707b6bf9c18e0f77c44). Thanks 😄